keanu-python/keanu/vertex/base.py

```diff
@@ -80,6 +80,12 @@ def get_label(self) -> Optional[str]:
         label = self.unwrap().getLabel()
         return None if label is None else label.getQualifiedName()
 
+    def get_label_without_outer_namespace(self) -> Optional[str]:
+        label = self.unwrap().getLabel()
+        if label is None:
+            return None
+        return label.withoutOuterNamespace().getQualifiedName()
```

Comment:
for consistency with the method above I'd prefer `return None if label is None else ...`